test-case-reporting/test/e2e.test.ts

```diff
@@ -32,7 +32,7 @@ jest.mock('../src/db', () => ({
     }),
 }));
 
-describe('end-to-end', () => {
+describe.skip('end-to-end', () => {
```

Comment:
?

Reply:
Skipping them because we don't expect them to pass at all yet -- would you prefer I not skip them?

Reply:
LGTM